### PR TITLE
Fix misspells

### PR DESCRIPTION
**Pull request by shendiaomo**

### CODE REVIEW

File: pkg/workflow/argo/fetch.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func (w *Workflow) Fetch(req *pb.FetchRequest) (*pb.FetchResponse, error) {
 	}
 
 	if req.StepPhase == "" {
-		// the 1-th container execute `argoexec wait` to wait the preiority step, so package the 2-th container's command code.
+		// the 1-th container execute `argoexec wait` to wait the priority step, so package the 2-th container's command code.
```
Comment (lhw362950217) [inline pkg/workflow/argo/fetch.go]: 1-th ->1st，2-th -> 2nd may be better

File: pkg/workflow/argo/fetch.go [hunk 2]
```diff
@@ -146,7 +146,7 @@ func (w *Workflow) Fetch(req *pb.FetchRequest) (*pb.FetchResponse, error) {
 
 	if isPodCompleted(pod) {
 		// TODO(yancey1989): add duration time for the eoeResponse
-		// eoe just used to simpler the client code which can be consistant with non-argo mode.
+		// eoe just used to simpler the client code which can be consistent with non-argo mode.
```
Comment (lhw362950217) [inline pkg/workflow/argo/fetch.go]: simpler -> simplify

**Review by lhw362950217 — COMMENTED** [review]
2 spelling check comments for u

**Review by weiguoz — APPROVED** [review]

**Review by lhw362950217 — COMMENTED** [review]
LGTM

**Pull request merged**